daemon/meson.build

```diff
@@ -1,12 +1,17 @@
 # Machine Learning Agent
 # This file should be included when ml-service feature is enabled.
 nns_ml_agent_incs = include_directories('includes')
-nns_ml_agent_srcs = files('main.c', 'modules.c', 'gdbus-util.c',
-  'service-db.cc', 'pipeline-dbus-impl.cc', 'model-dbus-impl.cc')
-if get_option('enable-tizen')
-  nns_ml_agent_srcs += files('pkg-mgr.c')
+__nns_ml_agent_srcs = ['main.c', 'modules.c', 'gdbus-util.c',
+  'service-db.cc', 'pipeline-dbus-impl.cc', 'model-dbus-impl.cc']
+if (get_option('enable-tizen'))
+  __nns_ml_agent_srcs += 'pkg-mgr.c'
 endif
 
+nns_ml_agent_srcs = []
+foreach f : __nns_ml_agent_srcs
+  nns_ml_agent_srcs += files(join_paths(meson.global_source_root(), 'daemon', f))
+endforeach
```

Comment:
This patch changes file path of object files.
Let's find another way.

origin: gdbus-util.o
w/ this change: _home_abuild_rpmbuild_BUILD_capi-machine-learning-inference-1.8.3_daemon_gdbus-util.o

refer: https://github.com/nnstreamer/api/pull/226/commits/051b34b0ff17d0ef6a2de904d423f62f026450b9